themes/digital.gov/layouts/partials/core/usa-collection.html

```diff
@@ -63,17 +62,22 @@
         {{ $href = .href }}
         {{ $title = .title }}
         {{ $summary = .summary }}
-        {{ $topic_icon = .topic_redirect }}
+        {{/* TODO: Need to lookup $href and return filepath to set $path for github editing */}}
       {{ end }}
```

Comment:
Need to write logic to output a filepath instead of a relative link when setting `top_resources` in front matter.